MotorDriver.h

```diff
@@ -0,0 +1,18 @@
+#ifndef MotorDriver
+#define MotorDriver
+
+#include <avr/io.h>
+#include <util/delay.h>
+
+class MotorDriver
+{
+  public:
+    MotorDriver();
+    void forward(int step);
+    void backward(int step);
```

Comment:
**Fix function name**
In motors, we generally use phrase like forward and backward but correct use of this is clockwise and counterclockwise.
You can never know as a library creator which way is front.